src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+    alt="New Relic Live archives"
+    src={NRLiveArchivesMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# Logs [#logs]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLogsMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic can help you manage your large volume of log data. We provide tools to ingest only valuable logs to cut costs. We offer a unified UI to connect your <InlinePopover type="logs" /> to your services. We also offer ways to organize them before you drown in them.
+
+  Learn more about logs [here](/docs/logs/get-started/get-started-log-management/).
+
+  </Side>
+  </SideBySide>
+
+# Network [#network]
+<SideBySide>
+  <Side>
+
+  With <InlinePopover type="networkMonitoring" />, you can analyze how well your routers, switches, and other networking devices perform. It will also build a visual network topology. This will let you find performance problems in locations. 
+  
+  Learn more about network [here](/docs/network-performance-monitoring/get-started/npm-introduction/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic network"
+  alt="New Relic network"
+  src={NRNetworkMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Mobile [#mobile]
+<SideBySide>
+  <Side>
+<img
+  title="New Relic mobile"
+  alt="New Relic mobile"
+  src={NRMobileMain}
+/>
+
+  </Side>
+  <Side>
+
+  The <InlinePopover type="mobile" /> capability sends telemetry directly from the end user's mobile app to New Relic. This allows us to analyze the app's performance on the user's device. 
+
+  Learn more about mobile [here](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# OpenTelemetry [#opentelemetry]
+<SideBySide>
+  <Side>
+
+  OpenTelemetry helps collect data from your applications and hosts, sending it to New Relic. New Relic's platform lets you analyze the data and fix any application problems. This toolkit simplifies the process of monitoring and troubleshooting your software. By utilizing OpenTelemetry, you can enhance the performance and reliability of your applications.
+
+  Learn more about OpenTelemetry [here](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic OpenTelemetry"
+  alt="New Relic OpenTelemetry"
+  src={NROpenTelemetryMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Serverless and Lambda [#serverless-lambda]
+<SideBySide>
+  <Side>
+
+<img
+  title="New Relic Lambda functions"
+  alt="New Relic Lambda functions"
+  src={NRLambdaMain}
+/>
+
+  </Side>
+  <Side>
+
+  We offer in-depth performance monitoring for your serverless AWS Lambda functions. Our Lambda layer gives one view into the details of your Lambda functions. You can understand what's happening in your serverless apps. Lambda monitoring is serverless. It uses CloudWatch data and code-level tools. They give more in-depth data. 
```

Comment:
Lambda monitoring is serverless. It uses CloudWatch data and code-level tools. They give more in-depth data. 
Comment

To:

Lambda monitoring is serverless and uses CloudWatch data and code-level tools to give you more in-depth data.